### PR TITLE
Adding netlify logo

### PR DESCRIPTION
**Pull request by liyokuna**
Changes:
- Adding netlify logo and size it to feet in the Footer

### CODE REVIEW

File: docs/app/app.component.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h1 class="sr-only">footer - site map & informations</h1>
                 <li class="nav-item"><span class="nav-link">v{{globals.ngBoostedVersion}}</span></li>
                 <li class="nav-item"><span class="nav-link">Code licensed under</span><a class="nav-link" href="https://github.com/Orange-OpenSource/Orange-Boosted-Angular/blob/master/LICENSE">MIT license</a></li>
                 <li class="nav-item"><span class="nav-link">Documentation licensed under</span><a class="nav-link" href="https://github.com/Orange-OpenSource/Orange-Boosted-Angular/blob/master/docs/LICENSE">CC BY 3.0</a></li>
-                <li class="nav-item"><a class="nav-link" href="https://ng-boosted.netlify.com">Ng-boosted Netlify</a> thanks to <a class="nav-link" href="https://www.netlify.com">Netlify</a></li>
+                <li class="nav-item"><a class="nav-link" href="https://ng-boosted.netlify.com">Ng-boosted Netlify</a> thanks to <a class="nav-link" href="https://www.netlify.com"><img src="https://www.netlify.com/img/global/badges/netlify-color-bg.svg" class="img-height"/></a></li>
```
Comment (Lausselloic) [inline docs/app/app.component.html]: Please add an alt text to the logo

File: docs/app/app.component.ts [hunk 2]
```diff
@@ -4,6 +4,11 @@ import { Globals } from './global';
 @Component({
     selector: 'app',
     template: require('./app.component.html'),
+    styles: [`
+    .img-height {
+        height: 1.4rem;
+    }
+    `]
```
Comment (Lausselloic) [inline docs/app/app.component.ts]: missing a comma, build failed

**Comment by liyokuna** [comment]
@Lausselloic  @PigeardSylvain tell me if it is ok .

**Comment by PigeardSylvain** [comment]
The picture is very small, I think we could add a title (title="Netlify"), and make the image bigger ?
image

Current size :+1: 
image

**Pull request merged**